composer.json

```
@@ -0,0 +1,48 @@
+{
+    "name": "ftw-soft/intercom-notification-channel",
```

Will need to be updated to:
```suggestion
    "name": "laravel-notification-channels/intercom",
```